notebooks/community/model_garden/model_garden_tfvision_image_segmentation.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<p>bullet lists not rendering correctly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>very strange. when i check the same way to write these contents in https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/model_garden/model_garden_tfvision_image_classification.ipynb, i think the format is reasonable. could you describe which part it wrong?</p>